beacon_node/beacon_chain/src/validator_monitor.rs

```diff
@@ -37,6 +40,9 @@ pub const HISTORIC_EPOCHS: usize = 10;
 /// Prometheus cardinality and log volume.
 pub const DEFAULT_INDIVIDUAL_TRACKING_THRESHOLD: usize = 64;
 
+/// Lag slots used in detecting missed blocks for the monitored validators
+pub const MISSED_BLOCK_LAG_SLOTS: usize = 4;
```

Comment:
Not a big fan of a const here. If we are using MinimalEthSpec for instance, I don't think we want the lag to be at 4. We might want to be different depending on the spec used.